ML_Model_Examples/Skill_of_Classifier.ipynb

```diff
@@ -0,0 +1,391 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "colab": {
+      "name": "Skill_of_Classifier.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "include_colab_link": true
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/pritampalit/ML4DQM/blob/master/Skill_of_Classifier.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SE49EHPK4zIL",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Using ROC curve, PR curve and Confusion matrix to evaluate the skill of Classifier\n",
+        "# PR curve is more effective in case of Imbalanced dataset\n",
+        "#Author : Pritam Palit, Shamik Ghosh & Francesco Fiori , 14/06/2020"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "vvMT6I6BHHN2",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import pandas as pd\n",
+        "#from google.colab import files\n",
+        "from google.colab import drive\n",
+        "drive.mount('/content/drive')\n",
+        "import matplotlib.pyplot as plt\n",
+        "import numpy as np\n",
+        "from ast import literal_eval\n",
+        "import json\n",
+        "#df= pd.read_csv('Tot_ZeroBias_UL2017_DataFrame_chargeInner_PXLayer_1.csv')\n",
+        "df= pd.read_csv('/content/drive/My Drive/Colab Notebooks/Tot_ZeroBias_UL2017_DataFrame_chargeInner_PXLayer_1.csv') # To read CSV in Google Colab\n",
```

Comment:
Please change to use EOS/SWAN instead, i.e. something like e.g.
/eos/project/c/cmsml4dc/ML_2020/UL2017_Data/
because it is what CMS members will use mostly.
Thanks

Reply:
This is an example input file for a particular type of tracker 1D histogram, which is not available in /eos/project/c/cmsml4dc/ML_2020/UL2017_Data/ , rather is created by an individual using https://github.com/cms-DQM/ML4DQM-DC_SharedTools/blob/master/DumpToCSV/DumpSingleHistogram.py of UL2017_Data/*.csv . You can also see similar example input in https://github.com/cms-DQM/ML4DQM-DC_SharedTools/blob/master/ML_Model_Examples/Double_AE_Step-by-Step.ipynb . But, I am changing the google colab portion and another small portion and recreating the PR. 
Thanks & regards